spec/index.bs

```diff
@@ -4225,6 +4397,22 @@ same module, they should avoid passing that information to
         </div>
 </dl>
 
+<div class="example">
+    Create a {{GPUShaderModule}} from WGSL code:
```

Comment:
I think it's worthwhile to include both a fragment and vertex entry point in this shader to give exposure to the idea of multiple entry points.